MEGA/Main.ino

```diff
@@ -47,15 +47,33 @@ State state();
       int baudRate = 9600;
 
     public:
+      char command;
       Comms() {
         Serial.begin(baudRate);
```

Comment:
We'll call `Serial.begin()` in the setup function once, no need to call it in the constructor for each subsystem class.